node_lambnik/scripts/update

```diff
@@ -21,9 +21,10 @@ then
 
         # Launch database container so we can populate it.
         docker-compose up -d database
+        docker-compose exec database bash -c "./updatedb"
```

Comment:
I don't seem to be able to run this the first time with success and have it load data.  After running `server` and building the containers, update does appear to reload data correctly.